sirepo/template/omega.py

```diff
@@ -285,12 +287,26 @@ def _report_info(sim_count, model_name, report_count):
                 for phase in range(_PHASE_PLOT_COUNT)
             ]
         )
+        if _is_genesis(run_dir, idx):
+            r.append(
+                [
+                    _report_info(idx + 1, "simFieldDistributionAnimation", 1),
+                ]
+            )
         idx += 1
         sim_dir = _sim_dir(run_dir, idx + 1)
 
     return res
 
 
+def _is_genesis(run_dir, index):
+    t, i = _sim_info(
```

Comment:
If you don't need `i` you can replace it with an underscore as a hint to the reader that it isn't used.

Reply:
Oh oops. I thought it was the radiasoft convention that we didn't do this.

Reply:
I guess I don't know about the convention. I do see it in a number of places...

Reply:
I agree with you that `_` should be the convention for an unused unpacked tuple variable